.circleci/config.yml

```diff
@@ -22,8 +29,38 @@ jobs:
       - run:
           name: Run lint
           command: make lint
+      - run: scripts/build-glean-metadata.py
+      - run: mkdir /tmp/workspace-data
+      - run: cp -r public/data/* /tmp/workspace-data
+      - persist_to_workspace:
+          root: /tmp/workspace-data
+          paths:
+            - ./*
+  upload:
+    # this job uploads the built artifacts to the GCS bucket
+    # used to serve https://dictionary.protosaur.dev/
+    docker:
+      - image: google/cloud-sdk
+    working_directory: /tmp
+    steps:
+      - attach_workspace:
+          at: /tmp/workspace
+      - attach_workspace:
+          at: /tmp/workspace-data
+      - run: cp -r /tmp/workspace-data/* /tmp/workspace/public/data
+      - gcs/gcs-rsync:
```

Comment:
How did you provision the service account for this?

Reply:
I'd recommend the importing the service account into https://github.com/mozilla-services/data-sandbox-terraform/blob/master/projects/glean-dictionary/project/main.tf if this was created in the GCP console.

Reply:
I'll file a PR